### PR TITLE
settings should account for process.env because _reasons_

### PR DESCRIPTION
**Pull request by moshmage**
The publicRuntimeConfig should
1. match the new settings
2. be intertwined with the settings
3. but overwritten if database settings exist

---

To achieve this, these changes were made

- add publicRuntimeConfig to settings
- add uri to closeBounty
- recreate public runtime config

effectively, if we save our public settings to the database nothing alters, otherwise it will use the (public) values we have in our `.env`

---

This branch is a in-between and should be only merged on 2.1 when we remove the settings database from existance. This was a messed up idea, and we have to revert it. my bad.

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -9,6 +9,68 @@ const publicRuntimeConfig = {
...
+    42: 'kovan',
+    3: 'ropsten',
+    4: 'rinkeby',
+    5: 'goerli',
+    1285: 'moonriver',
+    1337: 'localhost',
+    1500: 'seneca',
+    1501: 'afrodite',
+    1502: 'irene',
+    1503: 'iris'
```
Comment (vhcsilva) [inline next.config.js:73]:
```suggestion
    1503: 'iris',
    1504: 'diogenes',
    1505: 'aurelius'
```

File: next.config.js [hunk 2]
```diff
@@ -9,6 +9,68 @@ const publicRuntimeConfig = {
...
+    defaultToken: process.env.NEXT_PUBLIC_CURRENCY_ID,
+    conversionList: process.env.NEXT_PUBLIC_CURRENCY_VSLIST,
+  },
+  defaultNetworkConfig: {
+    networkName: process.env.NEXT_PUBLIC_DEFAULT_NETWORK_NAME,
+    allowCustomTokens: process.env.NEXT_PUBLIC_ALLOW_CUSTOM_TOKENS,
+    adminWalletAddress: process.env.NEXT_PUBLIC_ADMIN_WALLET_ADDRESS,
+  },
+  nftUri: process.env.NEXT_PUBLIC_NFT_URI,
+  web3ProviderConnection: process.env.NEXT_PUBLIC_WEB3_CONNECTION,
```
Comment (vhcsilva) [inline next.config.js:36]: We already have these envs in the urls property

**Review by vhcsilva — CHANGES_REQUESTED** [review]

File: next.config.js (hunk 2 above)
Comment (moshmage) [reply next.config.js:36]: forgot to remove them

**Review by vhcsilva — APPROVED** [review]

**Review by clarkjoao — APPROVED** [review]

**Comment by moshmage** [comment]
closing because outdated

**Pull request closed without merging**